.gitignore

```diff
@@ -1,6 +1,7 @@
 node_modules
 npm-debug.log
 dist/
+vendor
```

Comment:
This will be need to be added to the Directories section of the [distignore file](https://github.com/10up/insecure-content-warning/blob/ce4fc6673579b36dd1c21f7c7ae6ba681900c5bf/.distignore#L1). This is to make sure the wordpress.org copy of the plugin doesn't end up with a copy of phpcs in error.

Reply:
Fixed here: https://github.com/10up/insecure-content-warning/pull/127/commits/2272f08d2a5f92d58b1a09094603647337fedeea